src/pages/home.html

```diff
@@ -36,6 +36,335 @@
+                                                <img class="w-full h-full object-cover" src="../../assets/images/spx2-5.webp" alt="product-1" />
+                                            </a>
+                                            <a href="#" class="w-full h-full bg-gray-900 absolute top-0 left-0 bg-opacity-0 group-hover:bg-opacity-50">
+                                            </a>
+                                            <div class="group-hover:flex justify-center hidden absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 w-full">
+                                                <div class="flex justify-center items-center w-full">
+                                                    <button
+                                                        class="bg-main text-white hover:bg-white hover:text-main h-[45px] px-[10px] py-[5px] rounded-[20px] flex items-center justify-center mx-2">
+                                                        <i class="fas fa-shopping-cart me-[5px]"></i>
+                                                        <span>Cho vào giỏ hàng</span>
+                                                    </button>
+                                                    <button class="bg-white h-[45px] w-[45px] rounded-full">
+                                                        <i class="fas fa-search text-[20px] text-gray-500"></i>
+                                                    </button>
+                                                </div>
+                                            </div>
+                                            <div
+                                                class="absolute text-white bg-red-500 w-[60px] h-[60px] left-[20px] top-[20px] rounded-full leading-[60px] text-center">
+                                                <span>- 29%</span>
+                                            </div>
+                                        </div>
+                                        <div class="product-item__info p-[15px] text-center text-[16px] flex flex-col items-center">
+                                            <h3>
+                                                <a href="#"><span>Cây xương rồng</span></a>
+                                            </h3>
+                                            <div class="flex items-center">
+                                                <svg class="w-4 h-4 text-yellow-300 me-1" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 22 20">
+                                                    <path d="M20.924 7.625a1.523 1.523 0 0 0-1.238-1.044l-5.051-.734-2.259-4.577a1.534 1.534 0 0 0-2.752 0L7.365 5.847l-5.051.734A1.535 1.535 0 0 0 1.463 9.2l3.656 3.563-.863 5.031a1.532 1.532 0 0 0 2.226 1.616L11 17.033l4.518 2.375a1.534 1.534 0 0 0 2.226-1.617l-.863-5.03L20.537 9.2a1.523 1.523 0 0 0 .387-1.575Z"/>
+                                                </svg>
```

Comment:
```suggestion
                                                <img src="../../assets/images/image_name.svg" alt="Girl in a jacket" class="w-4 h-4 text-yellow-300 me-1" >
```
E đưa đoạn code < svg>...< /svg> này ra file hình ảnh (image_name.svg) vào thư mục assets/images nha
Sao đó gọi trong này bằng thẻ img